src/components/menu/submenu-block/submenu-block.d.ts

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+export interface SubmenuBlockProps {
+  menuType?: 'light' | 'dark';
+  size?: 'default' | 'large';
```

Comment:
`size` prop is not defined in `submenu-block.component.js`

Reply:
yep 🤦